### DISCUSSION TITLE
Issues mounting with host dir

### DISCUSSION THREAD

**Discussion by zephiowl** [itzg/docker-minecraft-server#1204] [Q&A] [answered]
sudo docker run -v HDD/game-hosting/mc-xvr:/data --name xvrtest --net=host -d -it -e EULA=TRUE -e MEMORY=5G  itzg/minecraft-server

docker: Error response from daemon: create HDD/game-hosting/mc-xvr: "HDD/game-hosting/mc-xvr" includes invalid characters for a local volume name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed. If you intended to pass a host directory, use absolute path.
See 'docker run --help'.

**Comment by itzg** [comment, marked as answer]
When using `-v` arguments with `docker run`, then the host directory has to be an absolute path. Docker compose files let you use relative paths, so I prefer using that to avoid the annoyance of `-v`
